packages/contracts/src/interfaces/IRelayRegistrar.sol

```diff
@@ -34,6 +34,11 @@ interface IRelayRegistrar {
      */
     function registerRelayServer(uint256 baseRelayFee, uint256 pctRelayFee, string calldata url) external;
 
+    /**
+     * @return the block number in which the contract has been deployed.
+     */
+    function creationBlock() external view returns (uint256);
```

Comment:
why not name it getCreationBlock ? 
I have a PR to change all other view items into a getter/setter model, so why add now more code to change later?

Reply:
Well, creation block cannot have a setter, but I can rename it if you want. Do you mean that I should make a variable private and only expose it through a method?

Reply:
making it a getter will make the underlying field separate, and then it can be immutable - which means its cost is reduced to zero..